examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -970,6 +978,49 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_channel_read_with_downgrade(&mut self) -> TestResult {
+        let public_key_label = confidentiality_label(web_assembly_module_signature_tag(
+            &base64::decode(PUBLIC_KEY_BASE64.as_bytes())
+                .expect("could not decode Base64 public key"),
+        ));
+
+        // Create backend node with a confidentiality label corresponding to [`PUBLIC_KEY_BASE64`].
+        let (backend_init_write, backend_init_read) =
+            oak::channel_create("Test", &public_key_label).expect("could not create channel");
+        oak::node_create(
```

Comment:
Is a back-end node required? It seems that just creating a channel with a more restrictive label (and writing to, reading from that single channel) should be sufficient for the test.

Reply:
Once reproducilbe builds are fixed, I think there is value in testing the ability to use the downgrade privilege. I just meant that it should be sufficient to have a single channel with the `public_key_label`. IFC should allow writing to it, but block reading from it unless the privilege is used. Using the backend node is not needed to just test `channel_read_with_downgrade`.